📚 DOCS: announce post

choldgraf opened the pull request:
This is an announcement post for our blog. The goal is to tell folks a bit more about the new build of jupyter book, explain what it does, and drive some interest to it.

Once we merge this in, I'll post the same thing to the Jupyter Blog.

What do folks think?

cc @AakashGfude @chrisjsewell @gregcaporaso @jstac @mmcky

chrisjsewell commented on docs/updates/2020-08-07-announce-book.md:
```diff
@@ -0,0 +1,108 @@
+
+*Note: this announcement is cross-posted between the [Jupyter Blog](https://blog.jupyter.org) and the [Executable Book Project updates blog](https://executablebooks.org/en/latest/updates/index.html)*
+
+Jupyter Book is an open source project for building beautiful, publication-quality books and documents from computational material. With this post, we're happy to announce that Jupyter Book has been re-written from the ground up, making it easier to install, faster to use, and able to create more complex publishing content in your books. It is now supported by [the Executable Book Project](https: ...
```
"from computational material" sounds a bit odd to me, what does it mean?
(note: in the "What next?" section you use "computational content")

Maybe something like:

Jupyter Book is an open source project for combining text and data into beautiful, publication-quality books and documents.

chrisjsewell replied:
"builds open source tools interactive" is there some words missing after tools? "builds open source tools for interactive"?

choldgraf replied:
I updated this so it includes "source material with computational content" per @jstac's suggestion above

chrisjsewell commented on docs/updates/2020-08-07-announce-book.md:
```diff
@@ -0,0 +1,108 @@
+
+You can also browse the [new Jupyter Book documentation](https://jupyterbook.org), or [check out the Jupyter Book GitHub repository](https://github.com/executablebooks/jupyter-book).
+
+**Note**: Jupyter Book is still in beta, so things are always changing. We'd love your feedback, ideas, and PRs for how to make the project better!
```
````suggestion
```{note}
Jupyter Book is still in beta, so things are always changing. We'd love your feedback, ideas, and PRs for how to make the project better!
```
````

chrisjsewell commented on docs/updates/2020-08-07-announce-book.md:
```diff
@@ -0,0 +1,108 @@
+
+The new Jupyter Book has the following main features (with links to the relevant documentation for each):
+
+* **[Write publication-quality content in markdown](https://jupyterbook.org/content-types/markdown)**. You can write in either Jupyter markdown, or in [MyST Markdown, an extended flavor of markdown for scientific publishing](https://jupyterbook.org/content/myst).  This includes support for rich syntax such as [citations and cross-references](https://jupyterbook.org/content/citations), [math and e ...
```
nitpick, but I don't feel that the main features (i.e. the text in bold) stands out very well from the rest of the text, especially due to all the other links, it all just kind of blurs into one. Perhaps have the subsequent text as sub-bullet points or something?

<img width="599" alt="image" src="https://user-images.githubusercontent.com/2997570/89692610-1abe5800-d904-11ea-999a-78f587312835.png">

choldgraf replied:
@chrisjsewell what do you think about the layout on the current jupyterbook.org home page? Is that better?

chrisjsewell replied:
https://github.com/executablebooks/jupyter-book/pull/864

choldgraf replied:
I've updated this so that it mimics the jupyterbook.org front page

chrisjsewell commented on docs/updates/2020-08-07-announce-book.md:
```diff
@@ -0,0 +1,108 @@
+
+### An enhanced flavor of markdown
+
+The biggest enhancement to Jupyter Book is support for the [MyST Markdown language](https://myst-parser.readthedocs.io/en/latest/). MyST is a flavor of markdown that implements all of the features of Sphinx, allowing you to write scientific publications in markdown. It draws inspiration from RMarkdown and the reStructuredText ecosystem of tools. Anything you can do in Sphinx, you can do with MyST ...
```
Include the acronym expansion somewhere? i.e. Markedly Structured Text (MyST)

chrisjsewell replied:
link to sphinx docs and/or mention what it is (e.g. the sphinx static site generator)?

chrisjsewell replied:
ah you do mention what sphinx is later on. Maybe better earlier but not sure

choldgraf replied:
added a bit more explanation of sphinx + a link, and added acronym expansion

chrisjsewell commented on docs/updates/2020-08-07-announce-book.md:
```diff
@@ -0,0 +1,108 @@
+
+### More book output types
+
+By leveraging Sphinx, Jupyter Book will be able to support more complex outputs than *just* an HTML website. For example, we are currently prototyping **[PDF Outputs](https://jupyterbook.org/advanced/pdf.html)**, both via HTML as well as via Latex. This gives Jupyter Book more flexibility to generate the right book for your usecase.
```
```suggestion
By leveraging Sphinx, Jupyter Book will be able to support more complex outputs than *just* an HTML website.
For example, we are currently prototyping **[PDF Outputs](https://jupyterbook.org/advanced/pdf.html)**, both via HTML as well as via LaTeX.
This gives Jupyter Book more flexibility to generate the right book for your use case.
```

chrisjsewell commented on docs/updates/2020-08-07-announce-book.md:
```diff
@@ -0,0 +1,108 @@
+
+The biggest change under-the-hood is that Jupyter Book now uses [the Sphinx documentation engine](https://www.sphinx-doc.org/en/master/) instead of Jekyll for building books. By leveraging the Sphinx ecosystem, Jupyter Book can more effectively build on top of community tools, and can contribute components back to the broader community.
+
+Instead of being a single repository, the  old Jupyter Book repository has now been separated into several modular tools. **Each of these tools can be used on their own in your Sphinx documentation**, and they can be coordinated together via Jupyter Book:
```
```suggestion
Instead of being a single repository, the old Jupyter Book repository has now been separated into several modular tools.
**Each of these tools can be used on their own in your Sphinx documentation**, and they can be coordinated together *via* Jupyter Book:
```

jstac commented on docs/updates/2020-08-07-announce-book.md:
```diff
@@ -0,0 +1,108 @@
+
+*Note: this announcement is cross-posted between the [Jupyter Blog](https://blog.jupyter.org) and the [Executable Book Project updates blog](https://executablebooks.org/en/latest/updates/index.html)*
+
+Jupyter Book is an open source project for building beautiful, publication-quality books and documents from computational material. With this post, we're happy to announce that Jupyter Book has been re-written from the ground up, making it easier to install, faster to use, and able to create more complex publishing content in your books. It is now supported by [the Executable Book Project](https: ...
```
change "books and documents" to "books, websites and other documents"?

jstac replied:
change "computational material" to "source material including computational content"?

chrisjsewell commented:
also one reason to write one line per sentence, is because review comments can then be made for a particular sentence, rather than the whole paragraph 😉

choldgraf commented:
hey all - I'll plan to address these issues in ~24 hours, and will plan to post on Monday 👍

choldgraf commented:
I think I've addressed all comments here, and have ported over a draft to the Jupyter Blog. Does it look OK to folks?

https://blog.jupyter.org/announcing-the-new-jupyter-book-cbf7aa8bc72e

I think we should plan to release it tomorrow (since the blog already had something posted today. That sound OK to everyone else?

chrisjsewell commented:
Yeh looks good to me 👍
